### PR TITLE
Feature/daily advices crud

### PR DESCRIPTION
**Pull request by marialungu**

### CODE REVIEW

**Comment by RafaelClaudiu** [comment]
Looks good :)

**Comment by RafaelClaudiu** [comment]
One thing though? Why are you using moment ? Without moment won't work ?

**Review by RafaelClaudiu — APPROVED** [review]

**Comment by marialungu** [comment]
@RafaelClaudiu the date picker from ant design wants moment

File: Backend/admin/package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "date-fns": "^2.11.1",
     "firebase": "^7.13.1",
     "lodash": "^4.17.15",
+    "moment": "^2.24.0",
```
Comment (razvanip) [inline Backend/admin/package.json:14]: i think we have `date-fns` already no need for 2.
`date-fns` much faster then moment

**Review by razvanip — APPROVED** [review]
@marialungu nice.

just replace `moment` with `date-fns`.

**Pull request merged**